Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.8.5-buster as base
+
+WORKDIR /src
+
+RUN pip install poetry
+
+COPY . /src/
+
+RUN poetry install
+
+EXPOSE 5000
+
+FROM base as production
+ENTRYPOINT ["gunicorn", "--bind", "0.0.0.0:5000", "wsgi:app"]
```

Comment:
Have you tested this? This fails because the `gunicorn` command is not recognised. It is only available within the poetry venv. 

Also, I expect this to be hosted on port 80 when in prodution.